### ISSUE TITLE
`apify run --purge` doesn't purge

### ISSUE THREAD

**Issue by jirimoravcik** [apify/apify-cli#545]
`apify run --purge` doesn't purge, even tried with `npx apify-cli@1.0.0-beta.42 run --purge`

How to reproduce?
Run:
```
apify create
? Name of your new actor: purge-test
? Choose the programming language of your new actor: Python
? Choose a template for your new actor. Detailed information about the template will be shown in the next step. BeautifulSoup + HTTPX
? Do you want to install the following template?
 BeautifulSoup + HTTPX:
```
```
cd purge-test
apify run --purge
```
but now if you run the following command again:
```
apify run --purge
```
it won't scrape anything, because the purge didn't happen.

**Comment by B4nan** [comment]
Hmm, does the python SDK purge by default? It should.

cc @vdusek

**Comment by vdusek** [comment]
> Hmm, does the python SDK purge by default? It should.

I don't think so. I am used to running `apify run --purge`.

**Comment by B4nan** [comment]
Hmm, this was the old sdk v2 behavior, I don't recall if python SDK was released before or after crawlee, but we changed this with crawlee and SDK v3 - the storages are purged by default unless you opt-out of it via env var (which the CLI sets via `--no-purge`). Another thing we need to fix with the upcoming major versions.

cc @vladfrangu, we should add the explicit purging for python projects (to both 0.x and 1.0) as it will take some time for the python tooling to catch up.
